binn/model/util.py

```diff
@@ -1,14 +1,18 @@
 import os
 import pandas as pd
 
-def load_reactome_db():
+def load_reactome_db(input_source = "uniprot"):
     """
     Load default Reactome mapping and pathways files.
     """
     base_dir = os.path.dirname(os.path.abspath(__file__))
 
-    mapping_path = os.path.join(base_dir, "../../data/uniprot_2_reactome_2025_01_14.txt")
-    pathways_path = os.path.join(base_dir, "../../data/reactome_pathways_relation_2025_01_14.txt")
+    if input_source == "uniprot":
```

Comment:
I don't think this will work when the code is packaged and built as a whl. It's needs to be included in the MANIFEST or in setup.py as package_data, or imported in some other way, like this:

https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package